console/src/components/Common/TableCommon/DragFoldTable.js

```diff
@@ -81,16 +91,74 @@ class DragFoldTable extends Component {
           {...this.props}
           data={data}
           columns={cols}
-          onFoldChange={newFolded =>
-            this.setState(p => {
-              return { folded: newFolded };
-            })
-          }
+          onFoldChange={newFolded => {
+            if (this.props.onCollapseChange) {
+              this.props.onCollapseChange(newFolded);
+            }
+            this.setState({ folded: newFolded });
+          }}
           folded={this.state.folded}
         />
       </div>
     );
   }
 }
 
+// return column width to fit all rows content
+export const getColWidth = (
+  header,
+  contentRows = [],
+  MAX_WIDTH = 600,
+  HEADER_PADDING = 62,
+  CONTENT_PADDING = 18,
+  HEADER_FONT = 'bold 16px Gudea',
```

Comment:
As these values are not relevant to any other function, they were defined as constants in the fn def itself and put into params so that they can be overridden if needed.

Reply:
Feel free to refactor

Reply:
@rikinsk my motivation for this comment was, because these constants are CSS properties, these values shouldn't be hard coded here if anywhere. Because CSS inconsistencies have a very explicit impact on the end user.